components/NavigationBar/src/MDCNavigationBar.h

```diff
@@ -225,6 +225,12 @@ IB_DESIGNABLE
  */
 @property(nonatomic) MDCNavigationBarTitleAlignment titleAlignment;
 
+
+/**
+ If YES, the title label will be autosized to fit a certain width. The default value is NO.
+ */
+@property(nonatomic) BOOL adjustsFontSizeToFitWidth;
```

Comment:
Should this property also apply to the leading/trailing buttons?  Maybe rename to `titleAdjustsFontSizeToFitWidth`?